src/pages/run-a-node-grants/index.tsx

```diff
@@ -0,0 +1,332 @@
+                  <PageText mb={6}>
+                    <Text as='i'>
+                      Starting on <Text as='strong'>June 7, 2023</Text>, we will begin to accept
+                      proposals.
+                    </Text>
+                  </PageText>
+
+                  <PageText mb={6}>
+                    <Text as='strong'>Deadline: July 20, 2023</Text>
+                  </PageText>
+
+                  <PageText mb={6}>
+                    <Text as='i'>
+                      All of the details you&apos;ll need to apply can be found below.
+                    </Text>
+                  </PageText>
+                </section>
+
+                <section id='introduction' ref={ref}>
+                  <PageSection mb={6}>Introduction</PageSection>
+
+                  <List>
+                    <ListItem>
+                      The Ethereum Foundation is launching a new funding round to encourage the
+                      diversification of Ethereum nodes.
+                    </ListItem>
+                    <ListItem>
+                      Running a node on Ethereum provides increased security, access, control, and
+                      flexibility and allows you to participate in and contribute to the Ethereum
+                      network actively.
+                    </ListItem>
+                    <ListItem>
+                      Nodes are a critical element of Ethereum, and we are looking for creative
+                      proposals from a diverse range of users looking to run a node. You may be
+                      based in a different geographical region; perhaps you want your university to
+                      use a node for research purposes - surprise us with your application.
+                    </ListItem>
+                    <ListItem>
+                      Selected applicants will receive either a{' '}
+                      <Link
+                        fontWeight={700}
+                        color='brand.orange.100'
+                        href='https://dappnode.com/'
+                        isExternal
+                        _hover={{ textDecoration: 'none' }}
+                      >
+                        Dappnode
+                      </Link>{' '}
+                      sent directly to you or a fixed allocated sum to reimburse the cost of the
+                      hardware.
+                    </ListItem>
+                  </List>
+                </section>
+
+                <section id='why-run-a-node' ref={ref2}>
+                  <PageSection mb={6}>Why Run a Node?</PageSection>
+
+                  <PageText mb={6}>
+                    We want you to be creative and demonstrate why you want to run a node for the
+                    Ethereum ecosystem. This list is to inspire you:
+                  </PageText>
+
+                  <List>
+                    <ListItem>
+                      Based in a remote location and want to test the node performance
```

Comment:
```suggestion
                      Based in a remote location and want to test the node performance.
```
Just adding period to keep consistent with rest of list